client/status/viewmodel/render.go

```diff
@@ -374,13 +374,15 @@ func renderReplicationReport(t *stringbuilder.B, rep *report.Report, history *by
 			eta = time.Duration((float64(expected)-float64(replicated))/float64(rate)) * time.Second
 		}
 
-		t.Write("Progress: ")
-		t.DrawBar(50, replicated, expected, changeCount)
-		t.Write(fmt.Sprintf(" %s / %s @ %s/s", ByteCountBinaryUint(replicated), ByteCountBinaryUint(expected), ByteCountBinary(rate)))
-		if eta != 0 {
-			t.Write(fmt.Sprintf(" (%s remaining)", humanizeDuration(eta)))
+		if latest.State != report.AttemptDone {
```

Comment:
States `AttemptFanOutError` and `AttemptPlanningError` are also terminal states where displaying a progress bar doesn't make sense.

Consider adding an `IsTerminal()` method to `AttemptState`

Here's the code that sets the state:

https://github.com/zrepl/zrepl/blob/b9250a41a25796969246710e55c972f24d1284d6/replication/driver/replication_driver.go#L698-L715

Reply:
I added the `IsTerminal` method. Nice catch